pkg/lib/cockpit-components-listing.jsx

```diff
@@ -304,34 +308,34 @@ var Listing = React.createClass({
             bodyClasses.push("listing-ct-wide");
         var headerClasses;
         var headerRow;
-        var selectableRows;
+        // var selectableRows;
         if (!this.props.children || this.props.children.length === 0) {
             headerClasses = "listing-ct-empty";
             headerRow = <tr><td>{this.props.emptyCaption}</td></tr>;
         } else if (this.props.columnTitles.length) {
+            /* probably not needed, ListingRow.selected prop should be set to 'false' by default
```

Comment:
What's your intention here? Let the tests run once, then delete this code? Or should this stay commented out? (subject to bitrot)? @mvollmer, WDYT?

Reply:
I intend to remove this. Reading through the code - I think it's not needed.
Tests passed on React, I expect same on react-lite but willing to see result first.
So putting needs-work back until it's clear.

Reply:
I have removed this commit from this PR.